docs/how_to/create_custom_map.rst

```diff
@@ -0,0 +1,266 @@
+To do this, you need to provide `sunpy.map.Map` with both the data array as well as appropriate metadata.
+The metadata informs `sunpy.map.Map` of the correct coordinate information associated with the data array and should be provided to `sunpy.map.Map` in the form of a header as a `dict` or `~sunpy.util.MetaDict`.
+See this :ref:`sphx_glr_generated_gallery_map_map_from_numpy_array.py` for a brief demonstration of generating a Map from a data array.
+
+The keys required for the header information follow the `FITS standard <https://fits.gsfc.nasa.gov/fits_dictionary.html>`__.
+sunpy provides a Map header helper function to assist in creating a header that contains the correct meta information.
+The utility function :func:`~sunpy.map.header_helper.make_fitswcs_header` will return a header with the appropriate FITS keywords once the Map data array and an `astropy.coordinates.SkyCoord` or `sunpy.coordinates.frames` is provided.
+All the metadata keywords that a Map will parse along with their description are listed in the :ref:`Meta Keywords Table` at the end of this page.
+
+:func:`~sunpy.map.header_helper.make_fitswcs_header` also takes optional keyword arguments including ``reference_pixel`` and ``scale`` that describe the pixel coordinate at the reference coordinate (defined by the `~astropy.coordinates.SkyCoord`) and the spatial scale of the pixels, respectively.
+Here's an example of creating a header from some generic data and an `astropy.coordinates.SkyCoord`:
+
+.. code-block:: python
+
+    >>> import numpy as np
+    >>> from astropy.coordinates import SkyCoord
+    >>> import astropy.units as u
+
+    >>> from sunpy.coordinates import frames
+    >>> from sunpy.map.header_helper import make_fitswcs_header
+
+    >>> data = np.arange(0, 100).reshape(10, 10)
+    >>> reference_coord = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime = '2013-10-28', observer = 'earth', frame = frames.Helioprojective)
+    >>> header = make_fitswcs_header(data, reference_coord)
+    >>> for key, value in header.items():
+    ...     print(f"{key}: {value}")
+    wcsaxes: 2
+    crpix1: 5.5
+    crpix2: 5.5
+    cdelt1: 1.0
+    cdelt2: 1.0
+    cunit1: arcsec
+    cunit2: arcsec
+    ctype1: HPLN-TAN
+    ctype2: HPLT-TAN
+    crval1: 0.0
+    crval2: 0.0
+    lonpole: 180.0
+    latpole: 0.0
+    mjdref: 0.0
+    date-obs: 2013-10-28T00:00:00.000
+    rsun_ref: 695700000.0
+    dsun_obs: 148644585949.49
+    hgln_obs: 0.0
+    hglt_obs: 4.7711570596394
+    naxis: 2
+    naxis1: 10
+    naxis2: 10
+    pc1_1: 1.0
+    pc1_2: -0.0
+    pc2_1: 0.0
+    pc2_2: 1.0
+    rsun_obs: 965.3829548285768
+
+From this we can see now that the function returned a `sunpy.util.MetaDict` that populated the standard FITS keywords with information provided by the passed `astropy.coordinates.SkyCoord`, and the data array.
+Since the ``reference_pixel`` and keywords were not passed in the example above, the values of ``crpix`` and ``cdelt`` were set to the default values.
```

Comment:
```suggestion
Since the ``reference_pixel`` and keywords were not passed in the example above, the values of ``crpix`` and ``cdelt`` were set to the default values, which is the centre of the data array.
```